test-data/unit/semanal-typeddict.test

```diff
@@ -46,8 +46,8 @@ MypyFile:1(
   ImportFrom:1(mypy_extensions, [TypedDict])
   ClassDef:2(
     A
-    BaseTypeExpr(
-      NameExpr(TypedDict [mypy_extensions.TypedDict]))
+    BaseType(
+      typing.Mapping[builtins.str, builtins.str])
```

Comment:
What's this change?

Reply:
A consequence of the semanal changes that replace the TypeInfo with the `replaced` version.